recipes-multimedia/rpidistro-ffmpeg/ffmpeg_%.bbappend

```diff
@@ -0,0 +1 @@
+PROVIDES:append = " virtual/ffmpeg"
```

Comment:
@EasyIP2023 Did you also propose this upstream?

Reply:
Here is upstream :) please call the recipe something like vlc-rpi and then let distros use PREFERRED_PROVIDER

Reply:
@kraj Check the MR. This is what is proposed here. It's just that we need the core to also align to provide that.

Reply:
submitted